src/_blog/cryptotls.md

```diff
@@ -0,0 +1,52 @@
+---
+tags:
+- quic
+- go
+- standard library
+- crypto/tls
+title: Adding a QUIC API for Go's standard library TLS package
+description:
+date: 2023-08-15
+permalink: "/2023-08-15-quic-crypto-tls/"
+translationKey: ''
+header_image: /quic-crypto-tls.png
+author: Marten Seemann
+---
+
+# Adding a QUIC API for Go's standard library TLS package
+
+You might have [heard us talk about this before](https://www.youtube.com/watch?v=aDHymXQJ4bs): QUIC has become the most important transport in libp2p. For example, in the IPFS network, QUIC connections account for 80-90% of all connections. [quic-go](https://github.com/quic-go/quic-go) is a QUIC implementation written in pure Go that is not only used in go-libp2p, but also in [Caddy](https://caddyserver.com/), a full-featured webserver (for its HTTP/3 support), [Adguard](https://github.com/AdguardTeam/AdGuardHome), for providing DNS over QUIC, and in the synchronization tool [syncthing](https://github.com/syncthing/syncthing/), and [many other projects](https://github.com/quic-go/quic-go#projects-using-quic-go).
```

Comment:
> in the IPFS network, QUIC connections account for 80-90% of all connections

1. Can we cite our source?  Maybe we can link to one of the new "public dashboards"?
2. Given the [broad definition of what IPFS is](https://specs.ipfs.tech/architecture/principles/), I think we should specify what we mean by the "IPFS network".  Maybe say "For example, QUIC accounts for 80-90% of the connections made to PL-run bootstrappers participating in the public IPFS DHT"

Reply:
Done.